pkg/apis/numaplane/v1alpha1/pipelinerollout_types.go

```diff
@@ -88,3 +88,8 @@ func (status *PipelineRolloutStatus) SetUpgradeInProgress(upgradeStrategy string
 func (status *PipelineRolloutStatus) ClearUpgradeInProgress() {
 	status.UpgradeInProgress = ""
 }
+
+// IsHealthy indicates whether the PipelineRollout is healthy.
+func (status *PipelineRolloutStatus) IsHealthy() bool {
+	return status.Phase == PhaseDeployed
```

Comment:
note that the way that the code is currently, we call it "Pending" if the user manually writes `desiredPhase=Paused` to their spec and the feature flag is turned on. Then this metric will deem it unhealthy. Perhaps need to reconsider if it should be "Pending" or "Deployed" in this case. 

The other effect of it being "Pending" is that it means ArgoCD would continually call it "Progressing" in the Jenkins pipeline. 

Let me write an issue for this.

Reply:
Actually, I take that back. I believe it should work but we should test it. Let me delete the comment from here, especially because I don't think it has an effect on the correctness of this.